src/archive/src/main.rs

```diff
@@ -634,6 +635,23 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
         stable_size() as f64,
         "Number of stable memory pages used by this canister.",
     )?;
+    w.encode_gauge(
+        "ii_archive_stable_memory_bytes",
```

Comment:
Is this useful? Do you need both pages and bytes?

Reply:
Yes, there was the agreement to expose the bytes and alarms will be created for us.

See the thread: https://dfinity.slack.com/archives/C063Z1ZBL8P/p1728300749385249?thread_ts=1727341770.248009&cid=C063Z1ZBL8P

Reply:
Ah, TIL. That makes sense 👍